source/module_cell/setup_nonlocal.h

```diff
@@ -17,8 +17,8 @@ class InfoNonlocal
 		Numerical_Nonlocal* Beta;/// nonlocal projectors (1-dimension array)
 		int *nproj; //mohan add 2010-12-19
 		int nprojmax; // mohan add 2010-03-07
-		double rcutmax_Beta;	//caoyu add 2021-05-24
-		const double& get_rcutmax_Beta(void) const { return rcutmax_Beta; }
+        double rcutmax_Beta = 0.0;
```

Comment:
The usual practice is to initialize the member variables of a class in its constructor.

Reply:
> The usual practice is to initialize the member variables of a class in its constructor.

Hi @dyzheng ,
What you said is the old-fashioned practice. In C++11, you don't have to do this in the constructor - "the basic idea for C++11 is to allow a non-static data member to be initialized where it is declared". Will you see https://www.stroustrup.com/C++11FAQ.html#member-init .

Reply:
@maki49 You are using space for indentation in a context using tab. Would you change it back?

Reply:
> > The usual practice is to initialize the member variables of a class in its constructor.
> 
> Hi @dyzheng , What you said is the old-fashioned practice. In C++11, you don't have to do this in the constructor - "the basic idea for C++11 is to allow a non-static data member to be initialized where it is declared". Will you see https://www.stroustrup.com/C++11FAQ.html#member-init .

I'm not saying it's wrong, but it's about keeping the format consistent， so please change the initialization format in this whole class, or keep the format consistant with before.

Reply:
> > > The usual practice is to initialize the member variables of a class in its constructor.
> > 
> > 
> > Hi @dyzheng , What you said is the old-fashioned practice. In C++11, you don't have to do this in the constructor - "the basic idea for C++11 is to allow a non-static data member to be initialized where it is declared". Will you see https://www.stroustrup.com/C++11FAQ.html#member-init .
> 
> I'm not saying it's wrong, but it's about keeping the format consistent， so please change the initialization format in this whole class, or keep the format consistant with before.

Then your "usual practice" means the practice in abacus... I see.

Reply:
OK. I've put the init in the constructor with tabs.